### DISCUSSION TITLE
[Feature] go-vod external transcoding / stream provider

### DISCUSSION THREAD

**Discussion by shperrung** [immich-app/immich#9012] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi!
Here is project that may be useful to be implemented as an option in Immich future releases.
https://github.com/m1k1o/go-transcode
It allows do not keep transcoded video and play them on demand with appropriate resolution, bitrate, format depending of network speed and browser capabilities.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#7666
